oci-library/oci-hol/java-management/understand-concepts-related-to-management-agent/understand-concepts-related-to-management-agent.md

```diff
@@ -0,0 +1,85 @@
+
+  ![image of response file](/../images/input-rsp-updated.png)
+
+## Task 2: Understand concepts related to Management Agent installation on OCI Compute Instance hosts
+
+Before the set up of the Management Agent, it is important to understand the concepts behind the Java Management Service:
+
+- **Java Management Service (JMS)**: A reporting and management infrastructure integrated with Oracle Cloud Infrastructure Platform services to observe and manage your use of Java SE (on-premises or in the Cloud). It enables you to observe and manage the use of Java in your enterprise.
+
+- **Management Agents**: Can be installed on a host to allow a service plug-in to collect data from the host where you installed the Management Agent. In the case of JMS, the management agent allows the **Java Usage Tracking service plugin** to track your use of Java.
+
+- **Oracle Cloud Agent (OCA)**: A lightweight process that manages **OCA plugins** running on compute instances. When you are using compute instances, you can deploy Management Agents by using the Oracle Cloud Agent.
+
+  As part of the set up of JMS on an OCI compute instance, the Management Agent will be deployed through the **Management Agent plugin** in OCA.
+  In addition, it is important to note that the **Oracle Java Management Service OCA plugin** is responsible for enabling Java runtime Lifecycle Management, which is an advanced feature of JMS. This workshop will focus on the basics of JMS, and thus focus on only the Management Agent OCA plugin. If you would like to learn more about the Lifecycle Management feature, you may refer to the [Java Lifecycle Management with Java Management Service](../../java-management-lifecycle-management/workshops/freetier/index.html?lab=introduction/introduction) workshop.
+  ![image of oca with management agent plugin](/../images/oca-plugins.png)
+
+- **Management Agent OCA plugin**: Helps to collect data from resources such as OSs, applications, and infrastructure resources for Oracle Cloud Infrastructure services that are integrated with Management Agent. Data can include observability, log, configuration, capacity, and health data.
+
+- **Java Usage Tracking service plugin**: A Service Plugin allows Management Agents to interact with data sources and send the data back to the cloud service. In the case of the Java Usage Tracking service plugin, data about Java Applications, Java Runtimes and Installations is collected from the host which can be either on-premises or from other cloud providers. It is important to note that this is a service plugin that is deployed to interact with the Management Agent, and is different from OCA plugins.
+  ![image of java usage tracking service plugin](/../images/java-usage-tracking-service-plugin.png)
+
+  You may now **proceed to the next lab.**
+
+## Want to Learn More?
+
```

Comment:
This is a good section to have, valuable. I am wondering if it can be enhanced. My initial confusion is because a few months ago I worked on this and many things have changed since then. 

Now, there is a clear distinction between OCI and non OCI hosts. If OCI host, you recommend then to use OCA method, and enable the OCA Java Management Service Plugin. On the Agent page, you just need JUT. 

If non OCI Host, rpm package to be installed, response file will need .jms and .jm for both JUT and LCM features. This will cause the Agent page to have both JUT and JMS plugin enabled. 

The gap I see is users with OCI hosts that choose not to use OCA OR (more likely) they did their installation in the past when the OCA method did not exist. Where does this leave them now? Perhaps can add that if they are using OCI host and installed via rpm package, then the instructions for non OCI hosts will work for them.    

Also, for OCA method, OCA side need to enable 1) Management Agent 2) Java Management Service + Agent side enable 1) JUT. For non OCI hosts, response file needs both .jms and .jm + Agent page needs 1) JUT 2) JMS. I don't know if it is just me, but as a user and I see the Agent Page having "Java Management Service" plugin, I might be curious why I don't need it. Might be good to add that this is specifically there because if you used the OCA method, enable OCA Java Management Service, you won't need to do it for Agent side. I think it is good to explain.